lib/vsc/administration/user.py

```diff
@@ -232,70 +230,81 @@ def pickle_path(self):
         a VscTier2AccountpageUser instance.
         """
         (path, _) = self.institute_path_templates[self.pickle_storage]['user'](self.account.vsc_id)
-        return os.path.join(self.institute_storage[self.pickle_storage].gpfs_mount_point, path)
+        return os.path.join(self.institute_storage[self.pickle_storage].backend_mount_point, path)
 
-    def _create_grouping_fileset(self, filesystem_name, path, fileset_name):
+    def _create_grouping_fileset(self, storage, path, fileset_name):
         """Create a fileset for a group of 100 user accounts
 
         - creates the fileset if it does not already exist
         """
-        self.gpfs.list_filesets()
+        fs_backend = load_storage_operator(storage)
+
+        try:
+            filesystem_name = storage.filesystem
+        except AttributeError:
+            logging.exception("Trying to access non-existent attribute 'filesystem' in the data storage instance")
+
+        try:
+            fs_backend.list_filesets()
+        except AttributeError:
+            logging.exception("Storage backend %s does not support listing filesets", storage.backend)
```

Comment:
no raise?

Reply:
done in https://github.com/hpcugent/vsc-administration/pull/134/commits/d9d2f11e38cfec377807ac56339b0498d1b1400d